AutoYaST/SLES-12-SPn-to-SLES-15-SP5/README.md

```diff
@@ -0,0 +1,17 @@
+This autoyast file has been used to upgrade from SLES12SP5 to SLES15SP5.
+The SLES12 system has only the mandatory repositories (modules) activated.
+
+Please do not change the kernel option according to the SUMA documentation.
+Disabling the self update feature in combination with self_update=0 pt.options=self_update and using a distribution based on GM (first official release) media DVD, will delete the installation rather then upgrading it.
+
+The autoupgrade option requires by default an registration server, however a SUMA server does not provided this expected functionality. Therefore the registration has to be turned off in the automation process. 
```

Comment:
Interesting that we don't have this in other upgrade profiles. Was this always the case or is this something new?

Reply:
This is the link to the documentation:
https://documentation.suse.com/sles/15-SP5/single-html/SLES-autoyast/#CreateProfile-upgrade
The last section is the registration, and if the registration is not disabled in the autoyast file, then a registration server will be contacted.
It was needed in my test environment.